test/test_DecisionTree.jl

```diff
@@ -26,7 +26,7 @@ end
 function test_DecisionTree()
     truth(x::Vector) = x[1] <= 0.5 ? -2 : (x[2] <= 0.3 ? 3 : 4)
     rng = Random.MersenneTwister(1234)
-    features = rand(rng, 10, 2)
+    features = rand(rng, 100, 2)
```

Comment:
By luck, the change from Julia 1.10 to 1.11 now means that `features `contained now points `x <= [0.5, 0.3]`, so we never trained a tree to predict `3`.